components/Document/DocumentViewer.tsx

```diff
@@ -300,6 +322,43 @@ const DocumentViewer = ({
   const shouldScroll =
     actualContentWidth > windowDimensions.width - LEFT_SIDEBAR_MAX_WIDTH;
 
+  const uploadPDF = (acceptedFiles: File[] | any[]): void => {
+    // const preload: Array<PreloadRow> = [];
+    // setLoading(true);
+    // acceptedFiles.map(() => {
+    //   const uuid = window.URL.createObjectURL(new Blob([])).substring(31);
+    //   preload.push({
+    //     citation_type: "LOADING",
+    //     id: uuid,
+    //     created: true,
+    //   });
+    // });
+
+    // setCreatedReferences(preload);
+    setUploadingPdf(true);
+
+    updateReferenceCitationFile({
+      payload: {
+        // TODO: calvinhlee - create utily functions to format these
+        citation_id: referenceItemDatum.id,
+        organization: currentOrg.id,
+        attachment: acceptedFiles[0],
+      },
+      onError: (e): void => {
+        console.log(e);
+        setUploadingPdf(false);
```

Comment:
On Error, can we put up an `alert("Upload failed. Try again later")` or something like that? It's an uncommon scenario but would be good to have the user feedback